code/modules/projectiles/projectile/reusable.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/projectile/bullet/reusable
+	name = "reusable bullet"
+	desc = "How do you even reuse a bullet?"
+	var/obj/item/ammo_casing/caseless/ammo_type = /obj/item/ammo_casing/caseless/
+
+/obj/item/projectile/bullet/reusable/on_hit(var/atom/target, var/blocked = 0)
+	if(!proj_hit)

```

Comment:
this is handled differently on TG...any idea why or what makes it different here?